test/core/integration/concrete/lockup/withdraw-max-and-transfer/withdrawMaxAndTransfer.t.sol

```diff
@@ -137,6 +137,7 @@ abstract contract WithdrawMaxAndTransfer_Integration_Concrete_Test is
         givenTransferableStream
         givenNotBurnedNFT
         givenNonZeroWithdrawableAmount
+        whenCallerRecipient(users.recipient)
```

Comment:
This modifier should not be needed as the name of the test is `test_WhenCallerCurrentRecipient`.